### ISSUE TITLE
Create Production check checklist page

### ISSUE THREAD

**Issue by naushinthomson** [elifesciences/schematron-gitbook#119]
- [x] Create page 
- [x] Add description of what is being covered (include pub checks and pub review)
- [x] Add description of checks required on this content, compiled from current protocol documents
- [x] Add examples covering correct and incorrect scenarios, as required to support checks
- [x] Review house style document to ensure all requirements are covered by GitBook
- [x] Include info about feature checks (including feature queries left in text)
- [x] Melissa approved page
- [x] Fred approved page
- [x] James approved page
- [x] Naushin approved page
- [x] Claudia approved page
- [x] Share page with Exeter and invite feedback

### Definition of done
- [x] Page completed
- [x] Team reviewed and approved page?
- [ ] Assign Fred to update schematron

**Comment by griffithsc** [comment]
Hi @naushinthomson, @JGilbert-eLife, @Melissa37, @FAtherden-eLife. This [page](https://elifesciences.gitbook.io/productionhowto/-M1eY9ikxECYR-0OcnGt/production-checklist) is now ready to review. Thanks in advance for looking over it!

**Comment by Melissa37** [comment]
>Production Checklist

title, C of checklist needs to be lower case

**Comment by Melissa37** [comment]
>standard for publication - this involves validating the XM

the dash should be an endash

**Comment by Melissa37** [comment]
>Articles that should be given priority at Publisher Review include:
Press articles
Feature content

Not necessarily press, as if it's press for 2 or 3 weeks time it could be left?
Also, I wonder whether this could be changed to prioritisation and list options:
- Features content
- Press for the following week

Can deprioritise the following if there is a lot of content to get through in one day and you know it will not all be got to:

- Press for a few weeks' time

- Articles with related content that is not yet in production

etc

**Comment by Melissa37** [comment]
>Production VS content processor

If we're going to give ourselves the status of being capilatilsed, we should do the same for other teams as well, like content processorts.

My preference would be to lower case them all, but that's me! We are not consistent anyway.

**Comment by Melissa37** [comment]
>Whilst carrying out a Publisher Review, if there are changes to be made they generally should be  communicated to the content processors via a Typesetter Query, as opposed to through the notes.

Could you show a screenshot of a Typesetter Query as well as a Note?

**Comment by Melissa37** [comment]
>then on the 'messages' tab and then the 'notes' tab, as illustrated below. 

In the screenshot Messages and Notes are capitalised, maybe the text should be too?

**Comment by Melissa37** [comment]
>Search the article reference number, look for the thread marked ‘Author Correspondence’, and check for any issues

Suggest changing to:

Search the Production inbox using the article reference number and look for the thread marked ‘Author Correspondence’. If there is any message from the author read this for actions they have asked for. 
Note: Sometimes authors may not respond to the proofing email so the title of the email thread may be non-standard or different. Be careful not to miss these

(maybe add some examples, like ejp numbers and emails?)

**Comment by Melissa37** [comment]
>This will be indicated in the 'custom' section, under the 'document details' and 'info' tabs.

Maybe change to:

>This will be indicated in the 'Custom' section, under the 'Document details' and 'Info' tabs.

To match the Kriya interface capitalisation

**Comment by Melissa37** [comment]
>As mentioned before, these articles should be prioritised at Publisher Review and the press date noted and inputted in the Publication History section within the article. 

I don't think the linked text goes to the right place.
I think they should only be prioritised if pub date is sooner
I'd change "The press date indicated should be added in the Publication History section within the article" to "and the press date noted and inputted in the Publication History section within the article"

**Comment by Melissa37** [comment]
>Some articles will be linked to other articles, indicating that they might need to be published together. 

Suggest changing to:
Some articles will be linked to other articles, indicating that they most likely need to be published together.

**Comment by Melissa37** [comment]
>If there is one or more linked article, they will be displayed in the 'related articles' section, under the 'document details' and 'info' tabs. Under 'article type', on the 'document details' page, you can see what kind of article it is.

Suggest capitalisation matches Kriya UI

**Comment by Melissa37** [comment]
> Red means it is not in the system yet
Yellow means that it is in progress in the system 
Green means it has been published

Suggest changing to:

Red means it is not in Kriya yet, it is still within EJP and not yet exported

Yellow means that it is in progress within Kriya 

Green means it has been published

**Comment by Melissa37** [comment]
>Sometimes the article that is displayed as linked is just a previously submitted, earlier version of the article. 

Suggest changing to:

Sometimes the article that is displayed as linked is just a previously submitted, earlier version of the article. Or a co submission that has been deleted.

**Comment by Melissa37** [comment]
>This can be checked by entering the number on EJP.

Suggest changing to:

This can be checked by searching the manuscript number on EJP.

Maybe give a screenshot?

**Comment by Melissa37** [comment]
>In order to assign an article to you, click on the relevant article in the 'Publisher Review' column. This will prompt a pop-up, displaying the 'Document details' for that article. You then click on the pen and paper button next to 'Assignee'. 

>This will generate a drop down menu, displaying the available users. Choose your name from the list. 

Suggest changing to:

In order to assign an article, click on the relevant article in the 'Publisher Review' column. This will prompt a pop-up, displaying the 'Document details' for that article. You then click on the pen and paper button next to 'Assignee'. 

This will generate a drop down menu, displaying the available users. Choose the person's name from the list.

**Comment by Melissa37** [comment]
>Title 
Check that the title makes sense.
Only italic formatting is allowed.
Check that it avoids unexplained acronyms. 
For latin words, the general rule is that genes, bacteria and species such as ‘S. pombe’ should be italicised, and that phrases such as ‘in vivo’ or ‘in vitro’ should not be italicised. However continuity between the title, impact statement and article content is most important, so please check that italics are consistent.
The capitalisation should be sentence case, but proper nouns can be capitalised. 

Are there Schematron checks for some of these? If so, should they be on this page?
We have a whole ticket in the backlog for Article title page. Would it contain more info than this? If not, should we add the schematron messages to this page and remove that ticket?

**Comment by Melissa37** [comment]
>Major subject areas
There should be at least one major subject area present. 

Should this be expanded on? Is there a maximum? What to do if there is not a MSA present?

**Comment by Melissa37** [comment]
>If an author has an ORCID ID, follow the link to ensure that it leads to the correct page (e.g. one with the same name as given in the manuscript).

I think we need to be careful about this - the point of ORCID is that it's an ID for a person who might display their name differently in different places...we don't want staff updating names based on ORCID

**Comment by Melissa37** [comment]
>If the 'corresponding author' box is ticked, their email address will be included in their details.

Should this read:

If the 'corresponding author' box is ticked, their email address should be present. Add if not.

**Comment by Melissa37** [comment]
>Check that they have at least one affiliation, but do not check the affiliations themselves, this will be done at the next stage.

What stage? I don't think the affiliations are manually checked?

**Comment by Melissa37** [comment]
>Author contributions will be listed here - check that they are formatted correctly and that any free-form contributions at the end are sensibly phrased and are not repetitive.

Can you provide examples of not sensibly phrased and repetitive?

**Comment by Melissa37** [comment]
>If an author contributed equally with another author, they will be linked. If the ‘Equal contributor with’ box is ticked, ensure that another name is tagged with it. 

I don't understand what this means :-(

**Comment by Melissa37** [comment]
>Any competing interests that an author may have, e.g. if they work as an editor for eLife or another journal, need to be listed here. If they have ticked the ‘None’ box, ensure that nothing is listed. 

I don't check any of this stuff as I rely on Schematron! Should the Schematron stuff be listed on this page?

**Comment by Melissa37** [comment]
>Sometimes affiliations need to be split up i.e., if there are two departments listed. A centre, within a department, within an institution, is okay. Two departments within an institution is not. Sometimes, authors belonging to two departments at the same institution will try to merge them into one affiliation, they need to be separated into two distinct affiliations. 

I did not think we checked this anymore and left as is?

**Comment by Melissa37** [comment]
>The received, accepted, PoA (if relevant) and preprint dates will be pre-filled from the submission system.
You don't need to check that these dates are correct at this point, they will be validated when you run the XML through the validator. Read about how to do this here. 

I think this should be changed to read:

The received, accepted, PoA (if relevant) and preprint dates will be pre-filled from the submission system.
You can check whether the article was PoD'd via the validator, see here.
You do not need to check the other history dates, but there is a Schematron message if there is a preprint date and please check the preprint site to confirm the date used is correct.

And provide instructions/screenshots for checking the history on BioRxiv.

Link to schematron messages?

**Comment by Melissa37** [comment]
>All funding received by authors in the process of researching and creating the article needs to be referenced here. Open each funding reference individually in order to check the details.

I don't think you need to do this, just rely on any schematron messages? And add the schematron messages to this page?

**Comment by Melissa37** [comment]
Crossref, not CrossRef :-)

**Comment by Melissa37** [comment]
>Follow this link and check that you are able to access the dataset

Suggest changing to:

Follow any links and check that you are able to access the dataset(s)

**Comment by Melissa37** [comment]
>Reviewers require a full name, but institution and country are optional. If a country is listed without an institution, remove the country.
For both sets of names, the same rules as for authors apply - names split properly, no full-stops after initials etc.

Is this not all covered by Schematron?

**Comment by Melissa37** [comment]
>When reviewing the changes, you can toggle between viewing the main article text with and without tracked  changes by clicking on the eye symbol at the top of the window.

There is a double space between tracked and changes

**Comment by Melissa37** [comment]
>Find out more about article level assets in their respective pages: figures and figure supplements, tables, additional files. 

Should match case of the title of these pages

**Comment by Melissa37** [comment]
>Check that all the journal references have DOIs. If a reference does not have a DOI, it will be highlighted with a red line. The DOI or any other missing information generally be found through a Google search

Not all journal references have a DOI so this would need rewording. Also, I don't think you need to check this, rely on the schematron and if schematron highlights any to check do it then. Otherwise don't bother - the schematron has preprogrammed to ignore ones we know don't have DOIs so it's a waste of time.

**Comment by Melissa37** [comment]
>We then validate the XML using our XML Validator.

I'd suggest this is the first step rather than the last step. Most of the checks are covered by the Schematron so the rest is noise as they say :-)

**Comment by Melissa37** [comment]
>You also need to click 'Validate Pub Date'. This will prompt a pop up which will notify you if any of these values are unexpected and need looking into. 

I'd expand this a bit to explain more fully

**Comment by Melissa37** [comment]
>Download PDFs
Download the PDFs and scroll down them, paying particular attention to the figures and tables, ensuring that there are no rendering issues.

Is it worth giving advice on this - only do this once everything else is done and don't wait around for it!! Do something else when this is being done :-)

**Comment by Melissa37** [comment]
>The majority of research content will skip this stage, that is, unless they require a higher copyediting level. If this is the case, Publisher Check will be carried out before it is sent out to an external copyeditor.

Suggest changing to:

The majority of research content will skip this stage, that is, unless the content processor has identified an issue they want the eLife staff to address pre-author (which is rare) or the article needs to be copy edited by a UK freelancer (because it requires more intervention). 

I don't think any checks should eb carried out by eLife staff before UK freelancers, just use this stage to assign it to someone.

**Comment by Melissa37** [comment]
>The article will come into the 'Publisher Check' section, and you will assign it to yourself as shown above. The process for carrying out a Publisher Check is then largely similar to the process for a Publisher Review, with the exception of Feature content, for which you will have to carry out some additional checks. You can find out how to carry out pre-author checks for feature content here. 
On completion of a Publisher Check, the content will not be sent to Continuum to be scheduled, and instead it will be sent to the typesetter (if there are edits to be made), to the author (if it is ready for them to review it), or to the 'waiting for assets' queue if it is waiting for a digest or decision letter. If a feature article is waiting for a decision letter, instead of sending it to the 'waiting for assets' queue, we put it on hold under our name. To do this, you locate the 'state' section under the 'document details' tab on the dashboard and change it to 'on hold'. You will be prompted to select a reason and input a comment. 

I'd rewrite this, and it's missing any info about how to assign it to a freelancer, which is the main purpose of this stage

**Comment by Melissa37** [comment]
Nice, thanks @griffithsc !!

**Comment by FAtherden-eLife** [comment]
> Search the article reference number, look for the thread marked ‘Author Correspondence’ ...

In addition to what Melissa has said above, it's worth noting that we need to search with the number followed by r (e.g. 70146r), in order for certain editorial/eJP emails to be returned in the search result.

**Comment by FAtherden-eLife** [comment]
> Grant references, however, are optional.

Should we change to Award IDs, in order to remain consistent with the editor interface/XML?

**Comment by FAtherden-eLife** [comment]
> Generated datasets mentioned in this statement should be included in the ‘Major Datasets - Generated’ field below.

Is it worth going into detail as to what needs doing if the authors have referred to a dataset in the DAS, but it's not added as a reference? Or link to a separate page if it's covered elsewhere?

**Comment by FAtherden-eLife** [comment]
> Impact Statement

The list on this page is different to the list [here](https://elifesciences.gitbook.io/productionhowto/-M1eY9ikxECYR-0OcnGt/article-details/content/article-structure/abstract-digest-impact-statement#what-needs-to-be-checked). Should they be the same?

**Comment by FAtherden-eLife** [comment]
> Editors and reviewers

Should we outline what should be done if there is no affiliation? 

Is it also worth mentioning that we also have Guest Editor's and these are usually the ones without affiliations?

**Comment by FAtherden-eLife** [comment]
> Download the PDFs and scroll down them, paying particular attention to the figures and tables, ensuring that there are no rendering issues.

Is there going to be a separate page on PDF layout etc., or should we go into more detail here?

**Comment by FAtherden-eLife** [comment]
> Signing off

Ready for publication needs explaining.

**Comment by FAtherden-eLife** [comment]
> XML

I agree with Melissa, XML is probably the first step (now that we're using Kriya 2). Is it worth adding a screenshot of a validated XML file as well?

**Comment by FAtherden-eLife** [comment]
> Queries

Can you explain what to do if you have a query for the authors at Pub Review?

If they asked for a second proof do you add it in Kriya (if so, how)? If they are not receiving a second proof, what's the best way to query them?

**Comment by FAtherden-eLife** [comment]
Does this page supersede the currently unused 'Production Check' page?

**Comment by FAtherden-eLife** [comment]
I don't know if this goes beyond the scope of this page, but currently we're just covering Pub Review and Pub Check. Where will we cover how to troubleshoot issues/ask author queries after/during Pub Review? 

Should this page also cover the Continuum Check? And what to do if an article has an insight etc.?

**Comment by FAtherden-eLife** [comment]
One more thing that just occurred to me. Under Production check, we should specify that the Pre version (instead of Final) of the Schematron should be used for validation.

**Comment by FAtherden-eLife** [comment]
> You also need to click 'Validate Pub Date'. This will prompt a pop up which will notify you if any of these values are unexpected and need looking into. 

Please can we add the caveat that this will not flag POAs that have been scheduled for Press (since that information isn't available anywhere in any API).

**Comment by naushinthomson** [comment]
>Whilst carrying out a Publisher Review, if there are changes to be made they generally should be communicated to the content processors via a Typesetter Query, as opposed to through the Notes. 

Can you indicate when a query might need to be added to the notes instead of as a typesetter query (i.e. if it's not at at a stage we can access)?

**Comment by naushinthomson** [comment]
>In this situation, it was necessary to add 'r' onto the end of the article number  

maybe add 'to find the email thread in the the inbox'

**Comment by naushinthomson** [comment]
>Sometimes the article that is displayed as linked is just a previously submitted, earlier version of the article. Or a co-submission that has been deleted. 

Change to 

>Sometimes the article that is displayed as linked is just a previously submitted, earlier version of the article, or a co-submission that has been deleted.

**Comment by naushinthomson** [comment]
>If the article appears to be a previously submitted, earlier version, or a co-submission that has been deleted, then the related article can be un-linked via the 'Related article' section in the Publisher Review.

I don't tend to use the method you outlined in the screenshots for checking linked articles but I don't know if it's just me. I search the number of the linked article (not the article you're working on) and if it says 'rejected' then I will delete it. Does that make sense? Maybe it would be clearer to show that process as at the moment the instructions don't indicate how to tell whether the article can be deleted.

**Comment by naushinthomson** [comment]
>If the 'Corresponding Email' box is ticked, their email address should be present. Add if not. 

Suggest changing to 
'Add from eJP if not.'

**Comment by naushinthomson** [comment]
In the Funding section, it would be worth linking to the [Funding](https://elifesciences.gitbook.io/productionhowto/-M1eY9ikxECYR-0OcnGt/article-details/content/funding-information) page 👍

**Comment by naushinthomson** [comment]
>Once the XML has been validated, you can perform more general checks on the article, as listed below. 

It might be worth mentioning that the following checks (from Title-Editors and reviewers) should only take a few minutes.

**Comment by naushinthomson** [comment]
>Download the PDFs and scroll down them, paying particular attention to the figures and tables, ensuring that there are no rendering issues. Aim to do another task while this is being done, because the generation process can take up to 10 minutes for each PDF file.

Suggest changing to
>Click on 'Export to PDF' to generate the article PDFs (aim to do another task at this point as the generation process can take up to 10 minutes). Scroll through the PDFs, paying particular attention to the figures and tables, ensuring that there are no rendering issues. Then, if the article has figure supplements, you will need to do the same for the figures PDF - click on 'Export to PDF (figures)' to generate the figures PDF.

**Comment by naushinthomson** [comment]
I don't think I have anything further to add, thanks for covering this!

**Comment by JGilbert-eLife** [comment]
I think the assigning articles part should come first? Since it'll be the earliest step.

**Comment by JGilbert-eLife** [comment]
I'm also not sure validating the XML should be first, since there may be changes required based on the author comments/queries that would affect issues in the XML. E.g. they may have provided reference details, citations, asked for changes etc. I usually make sure that stuff is sorted before checking the XML.

**Comment by JGilbert-eLife** [comment]
The Latin terms is a schematron check - it shouldn't be something we're doing by eye.

**Comment by JGilbert-eLife** [comment]
At this point, we do allow multiple departments in one affiliation - we've basically abandoned most of our rules around this excepting ensuring that states, postal details etc aren't included.

**Comment by JGilbert-eLife** [comment]
> Reviewers require a full name, but institution and country are optional. If a country is listed without an institution, remove the country.

Given how Kriya 2 works, probably more accurate to say, 'remove the affiliation.'

**Comment by JGilbert-eLife** [comment]
Probably worth stating that the Changes tab defaults to just showing the authors' edits.

**Comment by JGilbert-eLife** [comment]
> Just a general question about logging our time, in terms of aiming to do another task while the PDFs are generating on Kriya 2 do we want to capture this on Toggl? @JGilbert-eLife any thoughts?

Ideally, it should be tracked using Toggl, yes.

**Comment by JGilbert-eLife** [comment]
I don't think adding a VoR pub date when an article hasn't been PoAed is covered?

**Comment by JGilbert-eLife** [comment]
Thanks @griffithsc - I think the only other thing is moving the different checks into chronological order (e.g. Pub Check before Pub Review).
